raft/tracker/inflights.go

```diff
@@ -22,21 +22,26 @@ package tracker
 type Inflights struct {
 	// the starting index in the buffer
 	start int
-	// number of inflights in the buffer
-	count int
 
-	// the size of the buffer
-	size int
+	count int    // number of inflight messages in the buffer
+	bytes uint64 // number of inflight bytes
 
-	// buffer contains the index of the last entry
-	// inside one message.
+	size     int    // the max number of inflight messages
+	maxBytes uint64 // the max total byte size of inflight messages
+
+	// buffer contains the index of the last entry inside one message.
 	buffer []uint64
+	// size contains the corresponding byte size of the message.
+	sizes []uint64
```

Comment:
For allocation efficiency, this should better be paired with the index in a struct, and allocated in a single slice. Also, this would be less error-prone.

Reply:
Done.